README.md

```diff
@@ -287,6 +287,40 @@ parameters documented above also apply to AIX systems.
 * range (Parameter) - Sets the inter-physical volume allocation policy. AIX only
 * type (Parameter) - Configures the logical volume type. AIX only
 
+## Functions
+
+`lvm::bytes_to_size`: Converts a number of bytes to a size format that LVM can understand e.g. `lvm::bytes_to_size(214748364800)` would return `200g`
```

Comment:
Generally we do not document functions in the readme, this can be removed

Reply:
Why wouldn't we document them? There aren't documented anywhere else are they?